### PR TITLE
Add object schema in Kubernetes PSP policies

### PR DESCRIPTION
**Pull request by RamyasreeChakka**
Add object schema in Kubernetes PSP policies

### CODE REVIEW

File: Policies/Kubernetes/allowed-users/azurepolicy.parameters.json [hunk 1]
```diff
@@ -36,7 +36,30 @@
...
+                "max": {
+                  "type": "integer"
+                }
+              },
+              "required": ["min", "max"],
+              "additionalProperties": false
+            }
+          }
+        },
+        "required": ["ranges"],
```
Comment (EmandM) [inline Policies/Kubernetes/allowed-users/azurepolicy.parameters.json:60]: Ranges is not required on this policy. `MustRunAsNonRoot` doesn't require any values.

File: Policies/Kubernetes/fsgroup/azurepolicy.json [hunk 2]
```diff
@@ -47,7 +47,30 @@
...
+                    "max": {
+                      "type": "integer"
+                    }     
+                  },
+                  "required": ["min", "max"],
+                  "additionalProperties": false
+                }
+              }
+            },
+            "required": ["ranges"],
```
Comment (EmandM) [inline Policies/Kubernetes/fsgroup/azurepolicy.json:71]: "Ranges" is also not required here. `RunAsAny` doesn't require a range

File: Policies/Kubernetes/selinux/azurepolicy.json [hunk 3]
```diff
@@ -27,23 +27,43 @@
...
+                "type": "string"
+              },
+              "type": {
+                "type": "string"
+              },
+              "user": {
+                "type": "string"
+              }
+            },
+            "additionalProperties": false
```
Comment (EmandM) [inline Policies/Kubernetes/selinux/azurepolicy.json:59]: We're updating this policy to remove the parameters. Not sure if you want to pull these changes, or just update when I update the rego

File: Policies/Kubernetes/allowed-host-paths/azurepolicy.json [hunk 4]
```diff
@@ -27,20 +27,43 @@
...
+                    "readOnly": {
+                      "type": "boolean"
+                    }
+                  },
+                  "required": ["pathPrefix", "readOnly"],
+                  "additionalProperties": false
+                }
+              }
+            },
+            "required": ["paths"],
```
Comment (EmandM) [inline Policies/Kubernetes/allowed-host-paths/azurepolicy.json:59]: "paths" is not required as an empty list is used to block all (when we update the rego)

File: Policies/Kubernetes/selinux/azurepolicy.json (hunk 3 above)
Comment (RamyasreeChakka) [reply Policies/Kubernetes/selinux/azurepolicy.json:59]: You can update the policy again once your changes in Gatekeeper repo are approved.

**Pull request merged**